symfony/amazon-mailer/4.3/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+    "env": {
+        "#1": "MAILER_DSN=smtp://$AWS_ACCESS_KEY:$AWS_SECRET_KEY@ses",
+        "#2": "MAILER_DSN=http://$AWS_ACCESS_KEY:$AWS_SECRET_KEY@ses",
```

Comment:
Shouldn't we use `httpS` if possible?

Reply:
It **does** use httpS behind the scene, here `http` is not the "real" protocol, but merely a way to choose which interface to use.

Reply:
Ok, I didn't know this 👍